docs/helm-chart-troubleshooting.md

```diff
@@ -104,7 +104,31 @@ Requires any images referenced in a chart to be Red Hat Certified.
       match is not found the check fails.
     - if the image specified a tag value it is compared with the `repositories.tags.name` attributes. If a match is
       not found the check fails.
-- If the check fails use the point fo failure to determine how to address the issue. 
+- If the check fails use the point of failure to determine how to address the issue. 
+
+### `images-are-certified` v1.1
```

Comment:
I would just do this as this is what is changed in v.1.1 rather than repeat the entire section.